### PR TITLE
Update to gnatsd/README.md fixing authorization section 

### PR DESCRIPTION
**Pull request by gcolliso**
Update to gnatsd/README.md fixing authorization section addressing issue #662 "_INBOX.>" permission

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -577,7 +577,7 @@ Bob is REQUESTOR and can publish requests on subjects "req.foo" or "req.bar", an
 
 Joe has no permission grant and therefore inherits the default permission set. You set the inherited default permissions by assigning them to the `default_permissions` entry inside of the `authorization` configuration block.
 
-Note that `_INBOX.*` subscribe permissions must be granted in order to use the request APIs in Apcera supported clients. If an unauthorized client publishes or attempts to subscribe to a subject, the action fails and is logged at the server, and an error message is returned to the client.
+Note that `_INBOX.>` subscribe permissions must be granted in order to use the request APIs in Synadia supported clients. If an unauthorized client publishes or attempts to subscribe to a subject, the action fails and is logged at the server, and an error message is returned to the client.
```
Comment (derekcollison) [inline README.md]: This will be for all clients, whether official or not since its the server that enforces.

Comment (gcolliso) [reply README.md]: Does that mean you want the '...in Synadia Supported clients' removed or changed?

Comment (derekcollison) [reply README.md]: I think we can remove it, all clients will need the server to authorize publishing/subscribing to those subjects. Default mode of server is to allow all.

Comment (gcolliso) [reply README.md]: Should the entire note be removed or leave the sentence 'If an unauthorized client...'

Comment (derekcollison) [reply README.md]: I would remove and make sure we have section under authorization that talks about INBOXes specifically.

**Comment by derekcollison** [comment]
Resolves #662

**Pull request merged**